### ISSUE TITLE
Tests of new methods do not use the template

### ISSUE THREAD

**Issue by Fujiwaranosai** [SquaretestLLC/Squaretest#7]
Hi, I modified the template and if I generate a new test class, it works fine. However, if I add a new test method, it will generate with the default template, not my template.

**Comment by nate-dev** [comment]
Hi @Fujiwaranosai,

Thank you for the feedback.

Could you please confirm the following:

1. Are you using the latest Squaretest build (1.3)?
To check this, please go to: File | Settings | Plugins | Squaretest; the version (1.3 or 1.1.2, etc) should be displayed near the top of the right panel.  If you're using an older version, please install the update (Help | Check for Updates... | Update and Restart).

Squaretest was updated to use the template from the settings to generate completion suggestions in version 1.3. In prior versions, it only used internal templates.

2. Does your existing test class use a different test framework than the template in your settings?
If Squaretest determines that your existing test class uses a different framework than the template in your settings, it will use the default template for the framework used in the test class to generate completion suggestions; e.g. if your template uses JUnit5 but the existing test class uses JUnit4, Squaretest will use the default JUnit4 template to generate completion suggestions for the test methods.

**Comment by Fujiwaranosai** [comment]
Hi @nate-dev ,

1. I'm using the latest Squaretest build 1.3.
2. I changed the template in File | Settings | Squaretest. After saving, it changed the template name to <my-project>,java.ft. The test class I'm using Spring MockMvc. I generated the test class with Squaretest and it used my template. However, when I add a new test method, it doesn't use my template but the default one. So every time I want to add a new test method I have to delete and generate the test class again. I'm not sure how to make it work to add a new test method. Could you please help? Is it possible to do that? Regenerating the test class will lose all custom modification...

**Comment by nate-dev** [comment]
Thanks for the updates.
Are you able to add test methods using the "Generate Test Methods" menu option: Squaretest|Generate Test Methods or Alt+Insert+Generate Test Methods?
This should show a dialog that allows you to choose the template to use to generate the test methods. Does Squaretest show your template in the list of templates to choose from?

**Comment by Fujiwaranosai** [comment]
Yes if I do it from the menu, I can select my template and it works fine. I didn't know about the menu so I used the recommendation after typing a few letters and it doesn't use my template. Anyway now I know the right way to do. Thanks.
